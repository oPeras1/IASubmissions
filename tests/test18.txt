120	124	124	124	124	124	116	116	116	117	125	125	125	126	126	126	126	126	126	126	126	127	127	127	127	127	123	123	123	123
120	120	108	108	108	124	124	116	117	117	125	121	121	122	122	122	122	110	110	126	114	114	118	118	118	118	123	112	119	123
120	102	102	108	108	116	116	116	117	100	100	121	121	121	109	122	110	110	93	93	114	118	118	118	118	115	115	112	119	123
102	102	108	108	116	116	116	117	117	100	113	109	109	109	109	109	110	110	110	93	114	114	118	115	115	115	111	112	119	119
102	108	108	103	95	95	104	104	100	100	113	113	113	109	105	109	106	106	110	93	93	114	107	115	115	111	111	112	112	112
102	108	103	103	95	104	104	100	100	100	100	101	101	109	105	105	106	106	110	93	94	107	107	111	111	111	112	112	99	112
102	90	103	95	95	104	104	96	100	101	101	101	87	105	105	105	97	106	93	93	94	107	88	88	88	88	88	89	99	99
90	90	86	95	95	95	96	96	100	91	91	101	87	87	87	97	97	97	93	94	94	88	88	88	98	88	88	89	99	89
90	90	86	95	86	86	86	96	91	91	87	87	87	92	92	92	97	97	93	93	94	88	88	98	98	98	89	89	99	89
90	79	86	86	86	83	86	84	91	87	87	73	92	92	92	58	93	93	93	94	94	88	88	85	85	89	89	89	89	89
79	79	86	83	83	83	83	84	84	84	87	73	73	80	80	58	65	65	65	65	81	88	85	85	85	89	89	78	82	82
79	79	79	83	72	72	72	72	84	76	76	76	73	73	80	58	58	77	65	65	81	81	85	85	74	75	75	78	82	82
66	79	70	70	70	71	71	72	72	72	72	76	73	73	80	64	58	77	77	65	65	81	74	74	74	75	69	78	78	82
66	66	70	61	61	61	71	71	72	63	63	76	76	73	73	64	58	77	43	65	67	74	74	68	68	75	69	69	78	69
66	70	70	61	61	61	71	71	72	62	63	63	73	73	64	64	58	43	43	65	67	74	74	68	75	75	69	69	69	69
66	51	60	60	60	61	61	62	62	62	63	63	54	64	64	64	58	43	43	65	67	67	68	68	68	57	69	50	59	69
51	51	52	53	60	61	61	62	62	63	63	48	54	64	64	55	58	56	43	65	65	46	46	46	57	57	57	50	59	59
47	51	52	53	53	53	53	53	53	48	48	48	54	55	55	55	58	56	43	43	44	46	46	46	57	57	49	50	50	59
47	51	52	52	52	52	42	53	53	53	48	48	54	55	55	55	56	56	56	43	44	44	46	46	57	49	49	50	50	50
47	47	47	36	42	42	42	42	42	23	48	37	43	43	43	43	43	43	43	43	44	45	46	46	46	46	49	41	41	50
30	30	30	36	36	36	42	31	42	23	37	37	43	39	39	39	43	40	40	40	44	45	45	45	45	46	38	41	41	41
35	35	30	30	30	36	36	31	23	23	37	27	27	27	39	32	32	40	33	33	9	9	9	34	28	28	38	41	41	29
12	35	35	35	30	30	36	31	23	23	37	24	27	24	32	32	32	33	33	33	9	34	34	34	28	28	38	38	29	29
12	12	19	13	13	30	30	31	23	20	20	24	27	24	32	24	32	21	21	33	9	9	34	26	26	28	28	29	29	29
12	18	19	13	13	13	13	4	23	23	20	24	27	24	24	24	25	21	21	21	8	9	9	26	26	26	28	28	29	17
12	18	19	19	13	13	13	4	23	20	20	24	24	24	24	25	25	25	21	8	8	9	9	26	22	26	22	17	17	17
12	18	19	19	13	4	4	4	4	20	20	6	14	7	7	7	15	15	21	15	8	9	22	22	22	22	22	22	17	17
12	18	18	18	13	13	13	3	4	4	5	6	14	14	14	7	15	15	15	15	8	9	9	16	16	16	16	10	17	11
12	12	12	1	1	13	2	3	3	4	5	6	6	6	14	7	7	7	15	8	8	9	16	16	10	16	16	10	17	11
1	1	1	1	2	2	2	3	3	4	5	5	6	6	6	6	7	7	7	7	8	9	9	9	10	10	10	10	11	11