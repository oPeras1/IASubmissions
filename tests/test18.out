T	124	I	I	I	I	116	116	116	117	L	L	L	126	126	126	I	I	I	I	126	I	I	I	I	127	I	I	I	I
T	T	108	L	108	124	124	116	L	117	L	121	T	122	T	T	T	110	110	126	L	114	T	118	118	118	123	112	L	123
T	102	102	L	108	S	S	116	L	100	100	T	T	T	109	T	110	S	93	93	L	118	T	T	118	S	S	112	L	123
I	102	L	L	S	S	116	L	L	100	L	109	109	L	L	L	110	S	S	93	L	L	T	115	S	S	111	112	L	L
I	108	108	S	95	95	S	S	100	S	L	L	L	109	105	L	S	106	S	L	93	114	S	115	115	L	111	S	S	112
I	108	S	S	L	S	S	100	S	S	100	T	101	109	T	105	S	S	110	L	94	S	S	L	L	L	S	S	99	112
I	90	S	95	L	104	104	T	S	101	T	T	87	T	T	T	97	S	L	L	I	S	88	88	88	88	88	89	L	L
T	90	L	95	L	L	T	T	100	S	91	T	87	87	87	L	L	L	93	94	I	88	L	88	T	88	88	89	L	89
T	T	L	95	86	86	86	T	S	S	L	L	L	T	T	T	97	L	93	93	I	88	L	T	T	T	89	89	L	89
T	79	L	L	86	83	86	84	S	87	L	73	92	92	T	58	93	93	93	94	I	L	L	85	85	I	I	I	I	89
S	S	86	I	I	I	I	T	T	T	87	73	73	L	L	58	I	I	I	I	S	88	T	T	T	89	89	78	L	L
79	S	S	83	72	72	72	72	T	76	L	L	73	73	L	58	58	T	65	65	S	S	85	T	74	I	75	S	82	L
T	79	L	L	L	S	S	72	L	L	72	L	73	73	L	64	58	T	T	65	65	S	L	L	L	I	69	S	S	L
T	T	L	61	61	61	S	S	L	63	I	L	76	S	S	64	I	T	43	65	L	74	L	68	68	I	69	69	S	69
T	70	70	61	61	S	71	71	L	62	I	63	S	S	64	64	I	43	T	65	L	74	74	T	75	I	L	L	L	69
66	T	L	L	L	S	S	T	T	T	I	63	I	64	S	S	I	T	T	65	L	L	T	T	T	57	L	50	S	69
T	T	52	53	L	61	S	62	T	63	I	48	I	S	S	55	I	56	T	65	65	46	46	46	L	L	57	50	S	S
47	T	52	53	53	53	53	S	S	48	T	48	I	55	55	L	58	T	43	43	T	46	S	46	L	57	T	50	50	S
L	51	I	I	I	I	L	53	S	S	T	T	I	L	L	L	T	T	T	43	T	T	S	S	L	T	T	L	L	L
L	L	L	36	42	42	L	L	L	23	T	37	43	43	43	43	43	43	43	43	T	45	46	S	46	46	T	41	41	L
30	30	30	L	L	L	42	I	42	23	L	L	43	T	T	T	43	L	L	L	44	I	I	I	I	46	L	T	T	T
S	S	30	30	30	L	36	I	23	23	L	27	I	27	T	32	32	L	33	T	9	9	9	34	S	28	L	41	T	29
12	S	S	35	S	S	36	I	23	23	L	24	I	24	L	L	L	33	T	T	9	T	T	T	S	S	L	L	29	29
I	12	L	13	13	S	S	I	T	20	I	24	I	24	32	24	L	21	21	T	9	9	T	26	26	S	28	T	T	T
I	18	L	13	I	13	13	4	T	T	I	24	I	24	L	24	T	21	L	L	8	T	9	L	L	L	28	28	T	17
I	18	L	L	I	13	13	L	T	20	I	24	L	L	L	T	T	T	L	8	8	T	T	L	22	26	L	S	S	17
I	L	19	19	I	L	L	L	4	20	I	6	14	7	7	7	15	15	L	15	I	T	22	22	L	L	L	22	S	S
12	L	L	L	I	13	13	S	4	4	L	6	L	L	L	7	15	T	T	T	I	9	9	S	S	16	16	10	17	L
12	12	12	1	1	13	L	S	S	4	L	6	6	6	L	7	L	7	T	8	I	9	S	S	10	16	16	10	17	L
I	I	I	I	L	L	L	3	S	4	L	L	I	I	I	I	L	L	L	7	I	9	9	9	I	I	I	I	L	L